providers/softlayer/src/test/java/org/jclouds/softlayer/features/VirtualGuestApiLiveTest.java

```diff
@@ -125,6 +126,15 @@ public void testSetTagsOnVirtualGuest() throws Exception {
    }
 
    @Test(dependsOnMethods = "testSetTagsOnVirtualGuest")
+   public void testSetNotesOnVirtualGuest() throws Exception {
+      // Test with maximum allowed notes length - 1000 characters.
```

Comment:
Why the -1000? It would seem better to test the limit values, as those are the ones that may cause trouble.